src/blockrelay/graphene_set.h

```diff
@@ -184,7 +201,14 @@ class CGrapheneSet
             READWRITE(*pSetFilter);
         }
         if (!pSetIblt)
-            pSetIblt = std::make_shared<CIblt>(CIblt());
+        {
```

Comment:
This is the location of the code dup mentioned above.

Reply:
See above.